opcua_plugin/opcua.go

```
@@ -214,6 +223,9 @@ type OPCUAInput struct {
 	AutoReconnect                bool
 	ReconnectIntervalInSeconds   int
 	visited                      sync.Map
+	cachedCert                   []byte           // DER bytes or PEM
```

golangci-lint (1.62.2)

field `cachedCert` is unused

(unused)